src/util/CalculationTestHelpers.ts

```diff
@@ -120,3 +120,200 @@ export function getExampleValueSet(): fhir4.ValueSet {
     },
   };
 }
+
+/**
+ * Builds a statement_relevance map assuming that every single population in the population set is relevant.
+ * This is used by the logic view to determine which statements are unused to bin them properly. This is effectively
+ * creating a statement_relevance map that doesn't regard results at all, to be used when you don't have a result.
+ * @public
+ * @param {cqmMeasure} measure - The measure.
+ * @param {Population} populationSet - The populationSet we wish to get statement relevance for.
+ * @return {object} Statement relevance map for the population set.
+ */
+export function getStatementRelevanceForPopulationSet(
+  cqmMeasure,
+  populationSet
+) {
+  // create a population relevance map where every population is true.
+  const populationRelevance = {};
+
+  if (!_.isEmpty(populationSet[0]["observations"])) {
+    populationRelevance["observation_values"] = true;
+  }
+
+  // TODO can we assume a single populationSet?
```

Comment:
we can assume this is going to be called for each group. so yeah, one group at a time. no need to access with index or iterate over sets